### PR TITLE
1.1x Voting Power for Contributors

### PR DESCRIPTION
**Pull request by MegosAlpha**
Sorry for appearing to be hacking, I guess base64 and zlib was a bad idea. It's my first pull request ever, so that's why it probably looks messy in commit history. All contributors now receive an x1.1 bonus to voting perform, as a way of verifying accounts better.

### CODE REVIEW

**Comment by Alexays** [comment]
Before there was also the pr 297 :o

**Review by Alexays — COMMENTED** [review]
Wtf, why delete voters file ?

File: server/voters.html [hunk 1]
```diff
@@ -1,46 +0,0 @@
...
-                    for (var i = 0; i < list.length; i++) {
-                        tablehtml += "<tr><td>" + list[i].names + "</td><td>" + list[i].votes + "</tr>";
-                    }
-                    tablehtml += "</table>";
-                    result.innerHTML = tablehtml;
-                }
-            }
-        }
-    }
-</script>
```
Comment (phil-r) [inline server/voters.html]: why removing it?

File: settings.py [hunk 2]
```diff
@@ -71,6 +71,21 @@
 # used for calculating how long our voting window is
 TIMEZONE = "US/Pacific"
 
+# contains all members of the galactic federation (everyone against PR#322 at 10:33 MDT)
+# base64 and zlib applied
+FEDERATION = [
+    'phil-r',
```
Comment (phil-r) [inline settings.py]: well, I'm kinda proud of myself xD

File: settings.py [hunk 3]
```diff
@@ -71,6 +71,21 @@
 # used for calculating how long our voting window is
 TIMEZONE = "US/Pacific"
 
+# contains all members of the galactic federation (everyone against PR#322 at 10:33 MDT)
+# base64 and zlib applied
```
Comment (phil-r) [inline settings.py]: this line is not needed

**Comment by phil-r** [comment]
👎 Although my "name" is in the list, I don't think it's the right way to go. Changing voting power of different people is a tough decision, this can make those people with less voting power less motivated to contribute(except for smittyvb, he is just pain in the ass and deserves it)

**Comment by MegosAlpha** [comment]
I'll lower the power, and thanks for the advice. One sec...

File: server/voters.html (hunk 1 above)
Comment (MegosAlpha) [reply server/voters.html]: Accident, now fixed

**Comment by MegosAlpha** [comment]
@Leigende the file is back, I deleted it on accident. Also, voters of PR#297 have been added.

**Comment by yet-another-account** [comment]
I'd be almost okay with this if it was an insignificant advantage, like a voting power of 1.05 or something, but even then, I don't really like this idea.

**Comment by phil-r** [comment]
@eukaryote31 I had the same thoughts, Maybe all voters with `contributor` tag should have a bit more voting power(x1.05 like you said)? But I'm against hardcoded `federation` list for sure, sorry @MegosAlpha  😢

**Comment by MegosAlpha** [comment]
@phil-r Ok, I'll work on that. Thanks.

**Comment by o3LL** [comment]
How about making people doing "bad PR" getting less power instead of making an "Elite"?

File: settings.py (hunk 2 above)
Comment (rudehn) [reply settings.py]: You're the leader of the federation :D

**Comment by MegosAlpha** [comment]
I don't quite know how to check if a PR is really bad, because they always get revised. I think that if people do a good PR making them contributors, then it is better because this confirms them as a member of the Chaos community.

**Comment by MegosAlpha** [comment]
@phil-r @eukaryote31 I listened to your advice. It is now a 1.1x bonus to all contributors.

**Pull request closed without merging**